extensions/amp-access-scroll/0.1/scroll-component.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const TAG = 'amp-access-scroll-elt';
+import {CSS} from '../../../build/amp-access-scroll-0.1.css';
+import {Services} from '../../../src/services';
+import {createElementWithAttributes} from '../../../src/dom';
+import {installStylesForDoc} from '../../../src/style-installer';
+
+let init = false;
```

Comment:
This won't work very well because of multi-doc mode where the same binary can be used by several docs. Can you install the style in your access source? It's guaranteed to be a single source per document.

Reply:
Moved to scroll-impl constructor